Examples/ExamplesApp/Examples.swift

```diff
@@ -15,57 +15,70 @@
 import SwiftUI
 
 struct Examples: View {
-    /// The categories to display.
-    let categories = makeCategories()
-    
-    /// The category selected by the user.
-    @State private var selectedCategory: Category?
-    /// The example selected by the user.
-    @State private var selectedExample: Example?
```

Comment:
The reason to use list selection is so that selected list items will stay selected, even when the example's category is popped off the navigation stack and then returned to:

![Screen Recording 2025-04-08 at 8 49 07 AM](https://github.com/user-attachments/assets/079abedf-dc7f-4922-be82-b3dfd7b94706)

That would be preferred.

Reply:
No longer an issue with the DisclosureGroup change.